src/main/webapp/assets/modules/common/model/challengeRegistrant.js

```diff
@@ -125,6 +125,9 @@ techlooper.filter("challengeRegistrant", function (apiService, $rootScope, jsonV
     }
 
     registrant.recalculateSubmissions = function () {
+      _.each(registrant.submissions, function (submission) {
+        (submission.isRead == null) || (submission.isRead == undefined) ? submission.isRead = true : "";
```

Comment:
Line is too long.<br>Use '===' to compare with 'null'.<br>Use '===' to compare with 'undefined'.<br>Expected an assignment or function call and instead saw an expression.